relayer/chain/substrate/writer.go

```diff
@@ -110,7 +110,7 @@ func (wr *Writer) write(_ context.Context, c types.Call) error {
 
 	ext := types.NewExtrinsic(c)
 
-	era := types.ExtrinsicEra{IsMortalEra: false}
+	era := types.ExtrinsicEra{IsImmortalEra: true}
```

Comment:
why immortal?

Reply:
It was always immortal. I just changed the initialization to be more explicit about the fact that it is immortal.

I tried to make it mortal but it seems fairly complicated to do due to lack of docs. Anyway we do have a ticket for it.